core/extension/gdextension.cpp

```diff
@@ -762,24 +620,14 @@ GDExtensionInterfaceFunctionPtr GDExtension::get_interface_function(const String
 	return *function;
 }
 
-Error GDExtension::open_library(const String &p_path, const String &p_entry_symbol, Vector<SharedObject> *p_dependencies) {
-	String abs_path = ProjectSettings::get_singleton()->globalize_path(p_path);
+Error GDExtension::open_library(const String &p_path, const Ref<GDExtensionLoader> &p_loader) {
+	ERR_FAIL_NULL_V_MSG(p_loader, FAILED, "Can't open GDExtension without a loader.");
```

Comment:
Will `ERR_FAIL_NULL_V_MSG()` work with a `Ref<>`? I usually test `.is_null()`, but it's possible this works, and I just didn't know :-)

Reply:
Same as https://github.com/godotengine/godot/pull/91166#discussion_r1581577771. This should work because the macro just compares the `Ref<T>` with `nullptr`. We already use this macro with `Ref<T>` in the codebase, here's an example:

https://github.com/godotengine/godot/blob/e79e5bae956cda3ec0d9d0c7f431ff301ba97a8e/modules/mono/csharp_script.cpp#L2786

_`scr` is of type `Ref<CSharpScript>`._

Reply:
Ah, ok, thanks! I just didn't know that worked :-)